runtime/common/src/apis.rs

```diff
@@ -1008,13 +952,13 @@ macro_rules! impl_runtime_apis_plus_common {
 										location.clone().try_into().expect("convert to v3"),
 										i as u128
 									);
-									// set 1-1
-									<AssetManager as xcm_primitives::UnitsToWeightRatio<
+									// set 1-1 TODO: update before merging PR-2908
```

Comment:
We should uncomment this code. @librelois can we also check that the benchmarks are working?